Rocket.Chat/Controllers/Auth/AuthViewController.swift

```diff
@@ -47,6 +47,8 @@ final class AuthViewController: BaseViewController {
 
     @IBOutlet weak var activityIndicator: UIActivityIndicatorView!
 
+    @IBOutlet var accountCreationFields: [UIView]!
```

Comment:
What's `accountCreationFields`? Shouldn't it be just the `Register button`?

Reply:
Changed it to just the button